### PR TITLE
Fix terminology

### PR DESCRIPTION
**Pull request by smith558**
### Description

The original text seems to contain a minor mistake regarding the terminology. In CSS, `inline-size` and `block-size` are referred to as properties, not attributes. It's a common point of confusion.

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/the_box_model/index.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ The below diagram shows these layers:
 
 ### The standard CSS box model
 
-In the standard box model, if you give a box an `inline-size` and a `block-size` (or `width` and a `height`) attributes, this defines the inline-size and block-size (width and height in horizontal languages) of the _content box_. Any padding and border is then added to those dimensions to get the total size taken up by the box (see image below).
+In the standard box model, if you give a box an `inline-size` and a `block-size` (or `width` and a `height`) properties, this defines the inline-size and block-size (width and height in horizontal languages) of the _content box_. Any padding and border is then added to those dimensions to get the total size taken up by the box (see image below).
```
Comment (chrisdavidmills) [inline files/en-us/learn/css/building_blocks/the_box_model/index.md:125]:
```suggestion
In the standard box model, if you set `inline-size` and `block-size` (or `width` and `height`) property values on a box, these values define the `inline-size` and `block-size` (`width` and `height` in horizontal languages) of the _content box_. Any padding and borders are then added to those dimensions to get the total size taken up by the box (see the image below).
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Hi there @smith558, and thank you for fixing this. I agree with your change, however, I decided to take this opportunity to further improve the structure of this sentence — see my suggestion below.

If you like it, please feel free to apply it and we can get it merged.

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Comment by smith558** [comment]
> Hi there @smith558, and thank you for fixing this. I agree with your change, however, I decided to take this opportunity to further improve the structure of this sentence — see my suggestion below.
> 
> If you like it, please feel free to apply it and we can get it merged.

Looks good.

**Review by smith558 — COMMENTED** [review]
Looks good.

**Review by chrisdavidmills — APPROVED** [review]
@smith558 Awesome, let's get this merged. Thanks again!

**Pull request merged**